girder/notebooks/notebooks/__init__.py

```diff
@@ -11,6 +13,14 @@
 from .rest import Notebook
 
 def createNotebooks(event):
+
+    # If there is no current asset store, just return
+    try:
+        Assetstore().getCurrent()
+    except GirderException:
+        print('Warning: no current asset store. Notebook will not be created.')
```

Comment:
Can we use TerminalColor.warning(...) so we get some color and Warning => WARNING.

Reply:
Done.